docs/packages.md

```diff
@@ -56,7 +54,7 @@ A package source consists of a directory containing at least two files:
 Before you can build packages you need:
 - Docker version 19.03 or newer, which includes [buildx](https://docs.docker.com/buildx/working-with-buildx/)
 - If you are on a Mac you also need `docker-credential-osxkeychain.bin`, which comes with Docker for Mac.
-- `make`, `notary`, `base64`, `jq`, and `expect`
+- `make`, `base64`, `jq`, and `expect`
 - A *recent* version of `manifest-tool` which you can build with `make
```

Comment:
Do we still need this?

Reply:
No, we do not. manifest-tool now is embedded as a library inside the linuxkit binary. We no longer exec out to it.

For that matter, what to we use expect, base64 and jq for?

Reply:
kernel build still uses this